hooks/boot/20-certs_update.rb

```diff
@@ -18,6 +18,14 @@
   :default => false
 )
 app_option(
+  '--certs-reset',
+  :flag,
+  "This option will reset any custom certificates and use the self-signed CA " \
+  "instead. Note that any clients will need to be updated with the latest " \
+  "katello-ca-consumer RPM.",
```

Comment:
1. Yes:

After doing a `--certs-reset` on the Katello, you need to generate a new certs tarball and run the installer on the proxy, too.  After doing that the external proxy will work again.

2. No, you need to do the steps above